index.html

```diff
@@ -0,0 +1,343 @@
+                </button>
+              </li>
+              <li>
+                <ul class="language-list">
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+            </li>
+            <ul class="event-skill">
+              <li class="event18">
+                <h3>Frameworks</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+              <li class="event18">
+                <h3>Skills</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="consection">
+      <ul class="contain">
+        <li class="contact">
+          <h1 >Contact me</h1>
+          <p>
+            If you have an application you</br>
+            are interested in developing, a</br>
+            feature that you need built or a</br>
+            project that needs coding.</br>
+            I’d love to help with it!            
+          </p>
+          <form action="https://formspree.io/f/xayzkwne" class="form" method="post">
+            <li class="form-group">              
+              <input class="types"  type="text" rows="5" id="name" placeholder="Lorem Ipsum" maxlength="30" required/>
+            </li>
+            <li class="form-group">              
+              <input class="types" type="email" id="email" placeholder="loremipsum@gmail.com" required/>
+            </li>
+            <li class="form-group"> 
+             <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            </li>
+            <li class="form-group">
+              <input  value="Get in touch" type="submit" class="btn-input">
+            </li>
+          </form>
+        </li>
+      </ul> 
+        <div class="border-line">
+        </div>
+      </section>
+  </body>
```

Comment:
- [ ] [OPTIONAL] Please add the horizontal line at the bottom of the page to match the design from Figma 👀 
It is important to have keen eyes for details, even the littlest detail might make a huge difference in user experience. This is the time to practice 🏃🏽 ❤️